.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Update emacs-overlay
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
```suggestion
    - cron: '0 */4 * * *'
```

Reply:
Some random offset different from 0 0 might be useful to avoid running into schedules were everyone puts their action cron triggers.